src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2505,6 +2505,132 @@ public void TestUsingVarWithVarDeclarationTree()
             }
         }
 
+        [Fact]
+        public void TestAwaitUsingVarWithDeclaration()
```

Comment:
>TestAwaitUsingVarWithDeclaration [](start = 20, length = 32)

nit: I don't know if this test adds much compared to the one below (`TestAwaitUsingVarWithDeclarationTree`).

Reply:
Sure. A bunch of these are just moved over, but happy to clean up as we do so.